syncthing.if

```diff
@@ -0,0 +1,50 @@
+########################################
+## <summary>
+##	Role access for Syncthing
+## </summary>
+## <param name="role">
+##	<summary>
+##	Role allowed access
+##	</summary>
+## </param>
+## <param name="domain">
+##	<summary>
+##	User domain for the role
+##	</summary>
+## </param>
+#
+interface(`syncthing_role', `
+
+	gen_require(`
+		attribute_role syncthing_roles;
+		type syncthing_t, syncthing_exec_t, syncthing_config_home_t;
+	')
+
+	# Declarations
+	roleattribute $1 syncthing_roles;
+
+	# Policy
+	domtrans_pattern($2, syncthing_exec_t, syncthing_t)
+
+	syncthing_relabel_home_config_files($2)
```

Comment:
use this instead:

allow $2 syncthing_config_home_t:file { manage_file_perms relabel_file_perms };
allow $2 syncthing_config_home_t:dir { manage_dir_perms relabel_dir_perms };
allow $2 syncthing_config_home_t:lnk_file { manage_lnk_file_perms relabel_lnk_file_perms };